### ISSUE TITLE
Way to visually analyze url visits/clicks/feedback

### ISSUE THREAD

**Issue by cssherry** [spectrum-app/spectrum_backend#48]

**Comment by jgerminario** [comment]
Yeah this would be a great one for monitoring and investor pitch purposes. Probably a great task for once the main beta/launch stuff cools down since the data will still be coming in even without a great way to look at it

**Comment by jgerminario** [comment]
Reference Trello ticket for future definition of related tasks: https://trello.com/c/8mnLWeko/26-backend-monitoring-tools-launch-analytics

**Comment by jgerminario** [comment]
A small early thing that will help is to have some kind of a helper method enum map that categorizes Clicks based on the selector in a way that's easy for a data consumer to access. E.g. instead of the selectors it'll be easier to query if they clicked "Article link" or "Minimize icon". At a minimum before making the helper method we should have a comment in models.py that say what the selectors are.
